src/apps/menu/menu.tsx

```diff
@@ -157,89 +161,112 @@ const Menu: FC = () => {
+                  </div>
+                </div>
+                <div
+                  className="modal-header__name text-header-h4"
+                  data-cy="menu-patron-name"
+                >
+                  {userData?.patron?.name}
+                </div>
+                <Link
+                  href={menuViewYourProfileTextUrl}
+                  className="link-tag modal-header__link color-secondary-gray"
+                >
+                  {t("menuViewYourProfileText")}
+                </Link>
+              </div>
+              <div className="modal-profile__notifications">
+                {loansOverdue !== 0 && (
+                  <MenuNotification
+                    notificationNumber={loansOverdue}
+                    notificationText={t("menuNotificationLoansExpiredText")}
+                    notificationColor="danger"
+                    notificationLink={menuNotificationLoansExpiredUrl}
+                  />
+                )}
+                {loansSoonOverdue !== 0 && (
+                  <MenuNotification
+                    notificationNumber={loansSoonOverdue}
+                    notificationText={t(
+                      "menuNotificationLoansExpiringSoonText"
+                    )}
+                    notificationColor="warning"
+                    notificationLink={menuNotificationLoansExpiringSoonUrl}
+                  />
+                )}
+                {reservationsReadyForPickup !== 0 && (
+                  <MenuNotification
+                    notificationNumber={reservationsReadyForPickup}
+                    notificationText={t("menuNotificationReadyForPickupText")}
+                    notificationColor="info"
+                    notificationLink={menuNotificationReadyForPickupUrl}
+                  />
+                )}
+              </div>
+              <div className="modal-profile__container">
+                <div className="modal-profile__links">
+                  <div className="link-filters">
+                    <MenuNavigationList
+                      menuNavigationData={menuNavigationData}
+                      loansCount={loansCount}
+                      reservationCount={reservationCount}
+                      feeCount={feeCount}
+                    />
+                  </div>
+                </div>
+                <div className="modal-profile__btn-logout">
+                  <Link href={menuLogOutUrl}>
+                    <button
+                      type="button"
+                      className="btn-primary btn-filled btn-medium arrow__hover--right-small undefined"
+                    >
+                      {t("menuLogOutText")}
+                      <div className="ml-16">
+                        <ArrowIcon />
+                      </div>
+                    </button>
+                  </Link>
+                </div>
+              </div>
+            </div>
+          )}
+          {!isLoggedIn && (
```

Comment:
You can use [userIsAnonymous()](https://github.com/reload/dpl-react/blob/ead79b9764d29c711df2cba4b43661d08fc27c62/src/core/utils/helpers/user.ts#L3) to determine if the the user is authenticated.